src/Functional/monoid.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Widmogrod\Functional;
+
+use Widmogrod\FantasyLand\Monoid;
+use Widmogrod\FantasyLand\Semigroup;
+
+const mempty = 'Widmogrod\Functional\mempty';
+
+/**
+ * mempty :: a
+ *
+ * @param Monoid $a
+ * @return Monoid
+ */
+function mempty(Monoid $a)
+{
+    return $a->getEmpty();
+}
+
+const mappend = 'Widmogrod\Functional\mappend';
+
+/**
+ * mappend :: a -> a -> a
+ *
+ * @param Semigroup $a
+ * @param Semigroup $b
+ * @return Semigroup
+ */
+function mappend(Semigroup $a, Semigroup $b)
```

Comment:
@krtek4 WDYT about
- current `concat` do flattening o f`Foldable` then maybe rename it to `flatten` will make things more clear
- about `mempty` - there is no so much that I can do - `empty` is reserved word in PHP - do you have suggestion how to name it? maybe `identityOf`?